docs/fsharp/tutorials/asynchronous-and-concurrent-programming/async.md

```diff
@@ -67,7 +67,7 @@ In the example, the `printTotalFileBytes` function is of type `string -> Async<u
 
```

Comment:
not related to your PR but on line 66, could you also fix the - to a * to close the formatting on the word specification?